sem-reference_5b6968642c7d3a03f89d6bff.md

```diff
@@ -835,6 +841,22 @@ The previous command creates a new Semaphore 2.0 project that will be called
 Using `--repo-url` with `sem init` is much trickier because you should know
 what you are doing.
 
+### sem edit
+
+You can edit a project by using the `sem edit project` command, followed by the
+`name` of an existing project.
+
+``` bash
+sem edit project [name]
+```
+
+The `sem edit project` command will start your favorite text editor. In order
```

Comment:
I suggest using "default text editor", or "configured text editor" instead of "favorite text editor".